bukkit/pom.xml

```diff
@@ -58,7 +58,7 @@
         <dependency>
             <groupId>org.spigotmc</groupId>
             <artifactId>spigot-api</artifactId>
-            <version>1.8.8-R0.1-SNAPSHOT</version>
+            <version>1.12.2-R0.1-SNAPSHOT</version>
```

Comment:
Unless we are using 1.12.2 api features, do not bump, This will work perfectly fine with 1.12.2 servers without bumping. This ensures we are maintaining backwards compatibility.